src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -88,19 +103,19 @@ fabric-scoped data.</description>
       <arg name="Signature" type="OCTET_STRING"/>
     </command>
 
-    <command source="client" code="0x06" name="AddOpCert" optional="false">
+    <command source="client" code="0x06" name="AddNOC" optional="false">
       <description>Sender is requesting to add the new node operational certificate.</description>
       <arg name="NOCArray" type="OCTET_STRING"/>
       <arg name="IPKValue" type="OCTET_STRING"/>
       <arg name="CaseAdminNode" type="NODE_ID"/>
       <arg name="AdminVendorId" type="INT16U"/>
     </command>
 
-    <command source="server" code="0x08" name="OpCertResponse" optional="false">
-      <description>Response to AddOpCert or UpdateOpCert commands.</description>
+    <command source="server" code="0x08" name="NOCResponse" optional="false">
+      <description>Response to AddNOC or UpdateNOC commands.</description>
       <arg name="StatusCode" type="INT8U"/>
-      <arg name="FabricIndex" type="INT64U"/>
-      <arg name="DebugText" type="CHAR_STRING" length="128"/>
+      <arg name="FabricIndex" type="INT8U"/>
+      <arg name="DebugText" type="OCTET_STRING"/>
```

Comment:
Todo about making this CHAR_STRING?

Reply:
Yes, DebugText has to be a char string. I missed that changing. Do we still not have CHAR_STRING handling?

Reply:
It currently generates a `uint8_t *` for `CHAR_STRING`. There was some discussion about in Slack that triggered this change.

Reply:
We do not yet have CHAR_STRING handling in command fields, no.  https://github.com/project-chip/connectedhomeip/pull/7963 was going to do it but just got closed as stale...